### ISSUE TITLE
Tar error on macos

### ISSUE THREAD

**Issue by jpambrun** [kvaps/kubectl-build#2]
On macos I get:
```
tar: Option --record-size=100K is not supported
Usage:
  List:    tar -tf <archive-filename>
  Extract: tar -xf <archive-filename>
  Create:  tar -cf <archive-filename> [filenames...]
  Help:    tar --help
```

``` shell
❯ tar --version
bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
```

**Comment by jpambrun** [comment]
`--checkpoint` and  `--checkpoint-action` are also unavailable.

**Comment by kvaps** [comment]
Thank you for report!
If you remove these options, the rest code is working?

**Comment by jpambrun** [comment]
Not sure. The progress output is neat so I installed GNU tar from homebrew as a workaround.

**Comment by kvaps** [comment]
OK then I guess yes, the only thing we need is to add check for tar version

**Comment by kvaps** [comment]
fixed in `v1.3.0`, it would be nice if you could check with bsdtar on OSX

**Comment by jpambrun** [comment]
I just tested and it does work fine. However, you might end up looking at `If you don't see a command prompt, try pressing enter.` for a while which could be confusing. Perhaps `tar -v` would be best, but it could become noisy..
